cpp/ql/src/experimental/Best Practices/UselessTest.ql

```diff
@@ -0,0 +1,50 @@
+/**
+ * @name Useless Test
+ * @description Find any useless test of kind a==8 && a!=7
```

Comment:
Code must be enclosed in single quotes. See https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description.

Also, ideally the query description should describe the problem that this query is trying to highlight, and not what the query is doing. So I'd really rephrase this as something like:
```
A boolean condition that is guaranteed to never be evaluated should be deleted.
```